spec/models/item_spec.rb

```diff
@@ -1,7 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe Item, type: :model do
-  let(:item) {Fabricate(:item)}
+  let(:item) { item = Fabricate.build(:item)
+                item.categories << category
+                item.save
+                item}
```

Comment:
Expression at 7, 21 should be on its own line.<br>Inconsistent indentation detected.